joint_state_publisher/joint_state_publisher/joint_state_publisher

```diff
@@ -49,15 +49,25 @@ class JointStatePublisher():
                 if jtype == 'continuous':
                     minval = -pi
                     maxval = pi
+                    try:
+                        limit = child.getElementsByTagName('limit')[0]
+                    	minVel = 0
```

Comment:
Inconsistent indention level here, please use a consistent indention which matches the rest of the file.